src/placesview.cpp

```diff
@@ -471,7 +471,7 @@ void PlacesView::contextMenuEvent(QContextMenuEvent* event) {
         // Do not take the ownership of the menu since
         // it will be deleted with deleteLater() upon hidden.
         // This is possibly related to #145 - https://github.com/lxqt/pcmanfm-qt/issues/145
-        QMenu* menu = new QMenu();
```

Comment:
Just minor nitpick. Did you consider the above comment? Or if it is irrelevant as of today, then it probably needs to be removed to not confuse.

Reply:
Thank you so much for bringing this to my attention! I'll check that old problem soon.

Reply:
To be on the safe side, I just changed that comment.

The code is correct and has worked reliably. Because, in another place, I had the experience of avoiding a weird hang that was related to `GLib`, I prefer not to change the rest of this code.